From f79587df00edf64d8c2671dfce9368c3a848850d Mon Sep 17 00:00:00 2001
From: walteryang47 <walteryang47@users.noreply.github.com>
Date: Tue, 2 Aug 2016 21:43:34 +0800
Subject: [PATCH] Add find_multipaths to multipath.conf

So that local sata disks is not processed by multipath

Signed-off-by: walteryang <walteryang47@users.noreply.github.com>
---
 lib/vdsm/tool/configurators/multipath.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lib/vdsm/tool/configurators/multipath.py b/lib/vdsm/tool/configurators/multipath.py
index b40fd38..0e59cb1 100644
--- a/lib/vdsm/tool/configurators/multipath.py
+++ b/lib/vdsm/tool/configurators/multipath.py
@@ -40,6 +40,7 @@ _MPATH_CONF_DATA = """\
 %(current_tag)s
 
 defaults {
+    find_multipaths             yes
     polling_interval            5
     no_path_retry               fail
     user_friendly_names         no
-- 
1.9.3

